ktor-client/ktor-client-core/common/src/io/ktor/client/request/HttpRequestPipeline.kt

```diff
@@ -43,22 +43,27 @@ class HttpRequestPipeline : Pipeline<Any, HttpRequestBuilder>(Before, State, Tra
 /**
  * [HttpClient] Pipeline used for sending [HttpRequest] to remote server.
  */
-class HttpSendPipeline : Pipeline<Any, HttpRequestBuilder>(Before, State, Engine) {
+class HttpSendPipeline : Pipeline<Any, HttpRequestBuilder>(Before, State, Engine, Receive) {
```

Comment:
It looks like it is not just send pipeline anymore since we are also receiving a response as well